### PR TITLE
Add docs site, update readme

### PR DESCRIPTION
**Pull request by herojan**
Add docs site, update readme

### CODE REVIEW

File: docs-raw/installation/basic-install.yaml [hunk 1]
```diff
@@ -0,0 +1,291 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (irishshagua) [inline docs-raw/installation/basic-install.yaml]: Gonna be a PITA to keep this in synch with the actual CRD

**Comment by irishshagua** [comment]
too much stuff to review here... 🤷‍♂ 
Seems to be a lot of duplication now. If we merge, we're committing to keeping it all in synch, which will be difficult. Is there any way that we can share resources and maybe even automate the mkdocs step?

**Comment by herojan** [comment]
It's not that much, the internal doc features are being moved out here. We can reuse this for the internal site. Ignore the docs folder, that's generated. The basic-install is just bundling up a few files to give a oneliner install, I can get rid of it

**Comment by herojan** [comment]
Removed the bundled install to prevent sync issues, not sure about automating the site building since it's actually committed, would need some bot to be making commits

**Comment by irishshagua** [comment]
👍

**Comment by irishshagua** [comment]
👍

**Comment by herojan** [comment]
👍

**Pull request merged**